src/PublicApiGenerator/ApiGenerator.cs

```diff
@@ -721,10 +738,13 @@ static void AddPropertyToTypeDeclaration(CodeTypeDeclaration typeDeclaration, Pr
             var getterAttributes = member.GetMethod != null ? GetMethodAttributes(member.GetMethod) : 0;
             var setterAttributes = member.SetMethod != null ? GetMethodAttributes(member.SetMethod) : 0;
 
-            if (!HasVisiblePropertyMethod(getterAttributes) && !HasVisiblePropertyMethod(setterAttributes))
+            var hasGet = member.GetMethod is object && ShouldIncludeMember(getterAttributes);
+            var hasSet = member.SetMethod is object && ShouldIncludeMember(setterAttributes);
+
+            if (!(hasGet | hasSet))
```

Comment:
| or || ?

Reply:
Happy to change to suit your preference. My habit is `|` unless there is a chance of skipping any work. `||` is more to type and introduces a branch.

Reply:
It does not matter